examples/block-explorer/explorer-indexer/src/lib.rs

```diff
@@ -18,7 +18,7 @@
 //! assets from this example:
 //!
 //! ```bash
-//! cargo run --bin fuel-indexer -- --manifest examples/block-explorer/manifest.yaml
+//! cargo run --bin fuel-indexer -- --manifest examples/explorer-indexer/explorer_indexer.manifest.yaml
```

Comment:
Our `fuel-indexer` command requires a subcommand; in this case, that would be `run`. That's probably why you're encountering that error with the `--manifest` flag.

`cargo run --bin fuel-indexer -- run --manifest examples/explorer-indexer/explorer_indexer.manifest.yaml`

Reply:
noted down, thank you